<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartwindow</class>
 <widget class="QWidget" name="chartwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>649</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="one_day_button">
       <property name="text">
        <string>one day</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="two_days_button">
       <property name="text">
        <string>two days</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="three_days_button">
       <property name="text">
        <string>three days</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="week_button">
       <property name="text">
        <string>one week</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="month_button">
       <property name="text">
        <string>one month</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
